src/main/java/mekanism/client/render/obj/TransmitterBakedModel.java

```diff
@@ -1,5 +1,9 @@
 package mekanism.client.render.obj;
 
+import appeng.block.networking.CableBusBlock;
+import appeng.client.render.cablebus.CableBusRenderState;
+import appeng.client.render.cablebus.FacadeBuilder;
+import appeng.client.render.cablebus.FacadeRenderState;
```

Comment:
Only the `appeng.api` package is part of the public API. Everything else can change at any time.

Reply:
Yeah... I originally wanted to add this in a separate mod via mixins, but I thought it would be cool to have this native in Mekanism. If using the internals won't be accepted I could maybe change the PR to not use AE2, but then the facades would probably just be like a full block and not for individual sides.